.github/workflows/marketplace.yml

```diff
@@ -0,0 +1,22 @@
+name: "Action major version tag"
+
+on:
+  release:
+    types:
+      - published
```

Comment:
@bdougie we can get this workflow to trigger by changing the release `github.token` to admin level PAT but if the marketplace listing is unable to happen from semantic-release it's worth close to nothing to have half a workaround :D

Reply:
a better workaround is using `docker://` socket with ghcr.io, added those bits to the readme now